src/App.js

```diff
@@ -1,29 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import LandingPage from './components/LandingPage';
+import Reservations from './components/Reservations';
+import MyReservations from './components/MyReservations';
+import DeleteReservations from './components/DeleteReservations';
+import HomePage from './components/HomePage';
+import CourseDetails from './components/CourseDetails';
+import ReservationForm from './components/ReservationForm';
+import Authentication from './components/Authentication';
+import AddCourse from './components/AddCourse';
 
+const App = () => (
```

Comment:
- [ ] I can't find any links for adding or deleting courses:

![image](https://github.com/coderacheal/skill-habour-fe/assets/57421823/add1a326-5c87-4021-9428-a537a21c89fd)

If admin privileges are required, I also can't find in the schema a way to add admin users.

After some investigation, I found that those links are commented out:

![image](https://github.com/coderacheal/skill-habour-fe/assets/57421823/d08b541e-f649-49f6-b32c-96b4ea2c6597)

These are core requirements for a group of 3.  Kindly, add/finish those features. 👍